website/content/docs/connect/manage-traffic/route-to-local-upstreams.mdx

```diff
@@ -0,0 +1,113 @@
+
+   The parameters should match the values for regions and zones defined in your network. Refer to [`locality`](/consul/docs/services/configuration/services-configuration-reference#locality) in the services configuration reference for additional information.
+
+1. Verify that your service is also configured with a proxy. Refer to [Define service mesh proxy](/consul/docs/connect/proxies/deploy-sidecar-services#define-service-mesh-proxy) for additional information.
+Register or re-register the service to apply the configuration. Refer to [Register services and health checks](/consul/docs/services/usage/register-services-checks) for instructions.
+
+In the following example, the `web` service is available in the `us-west-1` region and `us-west-1a` zone on AWS:
+
+```hcl 
+service {
+  id  = "web"
+  locality = {
+    region = "us-west-1"
+    zone = "us-west-1a"
+  }
+}
+```
+
+## Enable service mesh proxies to route traffic locally
+
+You can configure the default routing behavior for all proxies in the mesh as well as configure the routing behavior for specific services.
+
+### Configure default routing behavior
+
+1. Configure the `PrioritizeByLocality` block in the proxy defaults configuration entry and specify the `failover` mode. This configuration enables proxies in the mesh to use the region and zone defined in the service configuration to route traffic. Refer to [`PrioritizeByLocality`](/consul/docs/connect/config-entries/proxy-defaults#prioritizebylocality) in the proxy defaults reference for details about the configuration.
+1. Apply the configuration by either calling the [`/config` HTTP API endpoint](/consul/api-docs/config) or running the [`consul config write` CLI command](/consul/commands/config/write). 
+
+   The following example writes a proxy defaults configuration entry from a local HCL file using the CLI:
+
+   ```shell-session
+   $ consul config write proxy-defaults.hcl
+   ```
+### Configure routing behavior for individual service
+
+1. Create a service resolver configuration entry and specify the following fields:
+   - `Name`: Specify the name of the service that the routing behavior defined in the configuration entry applies to.
+   - `PrioritizeByLocality`: This block enables proxies in the mesh to use the region and zone defined in the service configuration to route traffic. Set the `mode` inside the block to - `failover`. Refer to [`PrioritizeByLocality`](/consul/docs/connect/config-entries/service-resolver#prioritzebylocality) in the service resolver reference for details about the configuration. 
+1. Apply the configuration by either calling the [`/config` HTTP API endpoint](/consul/api-docs/config) or running the [`consul config write` CLI command](/consul/commands/config/write). 
+   The following example writes a proxy defaults configuration entry from a local HCL file using the CLI:
+
+   ```shell-session
+   $ consul config write web-resolver.hcl
+   ```
+
+## Start the proxies in your datacenter
```

Comment:
Given the advice to set client locality above (or defer to K8s/ECS), similar to @erichaberkorn 's comment above, I'm not certain we really need this section, since how to start and manage proxies is already documented in other pages, and assumed for a mesh deployment.